_posts/2022-04-19-arrow-r-cheatsheet.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: Arrow for R Cheatsheet
+date: "2022-04-19 00:00:00"
```

Comment:
We will want to update this + the filename right before we merge so that the publish date is right

Reply:
Updated to April 27, with the idea that we may merge on that date.